packages/zoe/test/swingsetTests/zoe/bootstrap.js

```diff
@@ -89,18 +89,21 @@ const makeVats = (E, log, vats, zoe, installations, startingExtents) => {
 
 function build(E, log) {
   const obj0 = {
-    async bootstrap(argv, vats) {
-      const zoe = await E(vats.zoe).getZoe();
-
+    async bootstrap(argv, vats, devices) {
+      const zoeBuilder = await E(vats.zoe).getZoe();
+      const vatAdminSvc = await E(vats.vatAdmin).createVatAdminService(
+        devices.vatAdmin,
+      );
+      const zoe = await E(zoeBuilder).buildZoeService(vatAdminSvc);
       const installations = {
-        automaticRefund: await E(zoe).install(automaticRefundBundle.bundle),
-        coveredCall: await E(zoe).install(coveredCallBundle.bundle),
-        publicAuction: await E(zoe).install(publicAuctionBundle.bundle),
-        atomicSwap: await E(zoe).install(atomicSwapBundle.bundle),
-        simpleExchange: await E(zoe).install(simpleExchangeBundle.bundle),
-        autoswap: await E(zoe).install(autoswapBundle.bundle),
-        sellItems: await E(zoe).install(sellItemsBundle.bundle),
-        mintAndSellNFT: await E(zoe).install(mintAndSellNFTBundle.bundle),
+        automaticRefund: await E(zoe).install(automaticRefundBundle),
```

Comment:
yay for removing `.bundle`, except I think maybe this was enabled by adding one inside `zoe.install`

Reply:
Yeah, when I fixed the mutliple levels in install, these ones came back.